decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -1,3 +1,8 @@
+<% content_for(:meta_description){ @proposal.body } %>
```

Comment:
It's not exactly the same as the block initializes the view context and leverages all rail's `capture` helpers and stuff. It makes sense performance-wise.

💥